src/ledger/TrustLineWrapper.cpp

```diff
@@ -377,6 +391,13 @@ TrustLineWrapper::IssuerImpl::isAuthorizedToMaintainLiabilities() const
     return true;
 }
 
+bool
+TrustLineWrapper::IssuerImpl::isClawbackEnabled() const
+{
+    // an issuer can't clawback from itself
+    return false;
```

Comment:
Should this be an internal error instead of `return false` ? It seems to me that it would just be a bug to reason about this flag on an issuer

Reply:
Yeah I think that's a good idea and it can be done with your suggested update to `CreateClaimableBalance`. Done.